runtime/testing-hydradx/src/lib.rs

```diff
@@ -729,16 +729,20 @@ impl Default for AssetLocation {
 	}
 }
 
+parameter_types! {
+	pub const SequentialIdStartAt: u32 = 1_000_000;
+}
+
```

Comment:
Please remove it from runtimes and rename `SequentialIdOffset` -> `SequentialIdStartAt` in the `common-runtime`, [here](https://github.com/galacticcouncil/HydraDX-node/blob/e7e515b9dea0e31fa6577f0df244826ff67ae845/runtime/common/src/lib.rs#L318)